weblab/entities/views.py

```diff
@@ -195,6 +196,21 @@ def get_redirect_url(self, *args, **kwargs):
         return reverse(url_name, args=[kwargs['pk'], 'latest'])
 
 
+class EntityDeleteView(UserPassesTestMixin, DeleteView):
+    """
+    Delete an entity
+    """
+    model = Entity
+    raise_exception = True
```

Comment:
It's part of `UserPassesTestMixin` which causes the view to raise a 403 error if the user doesn't pass the test rather than redirecting to login. Which seems to make sense, since deletion links aren't provided for those users anyway, it's not something that's likely to happen as part of the usual flow - a 'forbidden' response seems more appropriate (and they may already be logged in anyway). I'll add a comment.